tests/components/top_bar.test.ts

```diff
@@ -361,16 +360,16 @@ describe("TopBar component", () => {
   test("Can open a Topbar menu", async () => {
     const { app, parent } = await mountParent();
     expect(fixture.querySelectorAll(".o-menu")).toHaveLength(0);
-    const items = topbarMenuRegistry.getAll();
-    const number = items.filter((item) => item.children.length !== 0).length;
+    const items = topbarMenuRegistry.getMenu();
+    const number = items.filter((item) => item.children(parent.env).length !== 0).length;
     expect(fixture.querySelectorAll(".o-topbar-menu")).toHaveLength(number);
     triggerMouseEvent(".o-topbar-menu[data-id='file']", "click");
     await nextTick();
     expect(fixture.querySelectorAll(".o-menu")).toHaveLength(1);
-    const file = topbarMenuRegistry.get("file");
-    const numberChild = getMenuChildren(file, parent.env).filter(
-      (item) => item.children.length !== 0 || item.action
-    ).length;
+    const file = topbarMenuRegistry.getMenu().find((item) => item.id === "file")!;
```

Comment:
Can replace all the `registry.getMenu().find(...)` in the tests whith the helper `getNode(path, registry)`